### ISSUE TITLE
Feature Request: Continue processing after missing required values

### ISSUE THREAD

**Issue by hbobenicio** [kelseyhightower/envconfig#143]
With this spec:

```
type spec struct {
	Foo string `required:"true"`
	Bar string `required:"true"`
}
```

This process:
```
var s spec
err := envconfig.Process("myapp", &s)
if err != nil {
	log.Fatalln(err.Error())
}
```

stops processing right after the first missing required variable is detected

```
2019/03/13 10:42:54 required key MYAPP_FOO missing value
```

## Feature Request

It would be really useful in some cases (most? all of them?) just to keep processing to detect all missing required variables.

## Motivation

Just imagine an app that have a bunch o required variables. And then you fix one that is missing, then run the app again... then fix another one, then run it again... over and over, until you finally realize all the missing vars on your runtime configuration after some laborious run-and-fix work.

**Comment by teepark** [comment]
I agree that would be a significant DX improvement. If you want to backport the change in ndrewnee/envconfig#3 I can see merging that PR.

**Comment by TonyPythoneer** [comment]
Hi @teepark 
This issue seems to get solved. It should get closed.

**Comment by hbobenicio** [comment]
It doesn't seem to be fixed yet.
I'm still getting just one error message for the first missing required field.

```
$ go run main.go 
2019/12/16 09:07:08 required key MYAPP_FOO missing value
exit status 1
```

envconfig v1.4.0

**Comment by hbobenicio** [comment]
@teepark any insights here? the PR seems really simple. Can it be merged?
